### ISSUE TITLE
How to define "binaural" in .yaml?

### ISSUE THREAD

**Issue by WernerBleisteiner** [kickermeister/wav2bw64#3]
If I want to assign and define 2 channels correctly as "binaural" in an ADM - how to signal this in a yaml?

**Comment by kickermeister** [comment]
There is a basic implementation in a [feature branch](https://github.com/kickermeister/wav2bw64/tree/feature_binaural) which would allow you to do that by using "Binaural" as item type. Unfortunately, neither the EAR Production Suite nor the EBU ADM Renderer are currently supporting the Binaural typeDefinition, so I'm not sure how helpful such a feature would be at the moment.

**Comment by WernerBleisteiner** [comment]
Yep. As discussed. Will open a respective issue at [ebu_adm-renderer repo](https://github.com/ebu/ebu_adm_renderer)
